src/bit-systems/scene-loading.ts

```diff
@@ -81,7 +90,23 @@ function* loadScene(
     }
     environmentSystem.updateEnvironmentSettings(envSettings);
 
-    const sceneEl = AFRAME.scenes[0];
+    const cameraNode = (document.getElementById("scene-preview-node") as AElement).object3D!;
+    removeComponent(world, ScenePreviewCamera, cameraNode.eid!);
```

Comment:
This comment doesn't block this PR but is for future clean up.

Maybe using query to get camera node and its eid may be more consistent with other systems?

```
// in bit-components
export const ScenePreviewNode = defineComponent();

// in system
const query = defineQuery([ScenePreviewNode]);
// Assume there is a single entity having ScenePreviewNode component
const cameraNodeEid = query(world)[0];
const cameraNode = world.eid2obj.get(cameraNodeEid)!;
```